drivers/impedance-analyzer/ad5933/ad5933.h

```diff
@@ -189,4 +200,8 @@ double ad5933_calculate_impedance(struct ad5933_dev *dev,
 				  double gain_factor,
 				  uint8_t freq_function);
 
+void ad5933_set_settling_time(struct ad5933_dev *dev,
+			      uint8_t		mulitplier,
```

Comment:
I see some styling inconsistencies across the files, please:
```
cd no-OS
./ci/travis/astyle.sh
```
This will fix all style inconsistencies, make sure to add/commit the changes that the script did.